tests/datawidgetstests/test_autovizwidget.py

```diff
@@ -97,4 +102,5 @@ def test_convert_to_displayable_dataframe():
     assert_equals(result.dtypes[u'bool_col'], object)
     assert_equals(len(result[u'bool_col']), 2)
     assert_equals(result[u'bool_col'][0], 'True')
-    assert_equals(result[u'bool_col'][1], 'False')
\ No newline at end of file
+    assert_equals(result[u'bool_col'][1], 'False')
+    spark_events.emit_graph_render_event.assert_called_once_with(encoding.chart_type)
```

Comment:
I think this would this be a fantastic opportunity to start fresh on that method.